brave/brave-core
# Removes disconnected overlay from panel/page

Resolves https://github.com/brave/brave-browser/issues/5862

UI PR: https://github.com/brave/brave-ui/pull/531

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards and claim grant
- connect to uphold
 - click on wallet button in the panel/settings page
- disconnect
 - make sure that you see anon amount and claimed grants in the panel/settings page

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
